infra/vsts_agent_windows.tf

```diff
@@ -147,7 +148,7 @@ net stop winrm
 sc.exe config winrm start=auto
 net start winrm
 
-& choco install dotnetcore-2.1-sdk --no-progress --yes 2>&1 | %%{ "$_" }
+& choco install dotnetcore-${local.w[count.index].dotnet}.1-sdk --no-progress --yes 2>&1 | %%{ "$_" }
```

Comment:
Isn't the ".1" part of the version number? Seems odd to extract the major version but keep the minor version here.

Reply:
We're going from 2.1 to 3.1, and I guess I got lazy. This is a temporary state of affairs, so it did not seem worth doing any more than necessary for this one change.

Reply:
I also seem to have gotten forgetful. Thanks for pointing this out.